soh/src/overlays/actors/ovl_En_Ex_Item/z_en_ex_item.c

```diff
@@ -524,6 +524,8 @@ void EnExItem_DrawItems(EnExItem* this, GlobalContext* globalCtx) {
         }
 
         EnItem00_CustomItemsParticles(&this->actor, globalCtx, randoGetItem);
+        GetItemEntry_Draw(globalCtx, randoGetItem);
```

Comment:
how was this working without this call before?

Reply:
Oh didn't see this comment, this replaced a call to `GetItem_Draw` which worked fine for the drawing functions already in the game, but I wrote this new function instead that first checks for a pointer to a custom drawing function on the getItemEntry, uses that if present, or falls back to `GetItem_Draw` if not.